src/js/jwt/jwt.js

```diff
@@ -72,13 +73,33 @@ priv.decodeToken = (str) => {
 
     str = (str + '===').slice(0, str.length + (str.length % 4));
     str = str.replace(/-/g, '+').replace(/_/g, '/');
-
     str = decodeURIComponent(escape(atob(str)));
-
     str = JSON.parse(str);
+
     return str;
 };
 
+pub.doOffline = (key, val) => {
+    const url = urijs(window.location.href);
+    url.removeSearch(key);
+    url.addSearch(key, val);
+
+    const options = {
+        realm: 'redhat-external',
```

Comment:
Can you export [src/js/auth.js#L7~options](https://github.com/RedHatInsights/insights-chrome/blob/bc0725494efc0adc38059dd6bd19a7911d17de54/src/js/auth.js#L7) const and use it here as well? So we don't have to keep track of the same vars in two places.